contracts/exchange-rate-precompile/README.md

```diff
@@ -0,0 +1,9 @@
+## Exchange Rate Precompile methods
+
+The following table highlights the coded Exchange rate precompile methods and their availbility in the Hedera network exposed via the `0x168` address
+
+| Function Name                     | hash          | Consensus Node Release Version | HIP                              | Method Interface |
+|-----------------------------------|---------------|---------------|---------------------------------------------------|---------------------------------|
+| `tinycentsToTinybars`             | `0x2e3cff6a`  | [0.26](https://docs.hedera.com/hedera/networks/release-notes/services#v0.26)  | [HIP 475](https://hips.hedera.com/hip/hip-475)                                               |`tinycentsToTinybars(uint256)` |
+| `tinybarsToTinycents(uint256)`    | `0x43a88229`  | [0.26](https://docs.hedera.com/hedera/networks/release-notes/services#v0.26)  | [HIP 475](https://hips.hedera.com/hip/hip-475)                                               |`tinycentsToTinybars(uint256)` |
```

Comment:
probably tinybarsToTinycents for Function Name and tinybarsToTinycents(uint256) for method interface?

Reply:
Fixed